base/operators.jl

```diff
@@ -154,15 +154,17 @@ Values that are normally unordered, such as `NaN`,
 are ordered after regular values.
 [`missing`](@ref) values are ordered last.
 
-This is the default comparison used by [`sort`](@ref).
+This is the default comparison used by [`sort!`](@ref).
 
 # Implementation
 Non-numeric types with a total order should implement this function.
 Numeric types only need to implement it if they have special values such as `NaN`.
 Types with a partial order should implement [`<`](@ref).
-See the documentation on [Alternate orderings](@ref) for how to define alternate
+See the documentation on [Alternate Orderings](@ref) for how to define alternate
 ordering methods that can be used in sorting and related functions.
 
+See also [`isequal`](@ref), [`isunordered`](@ref).
+
```

Comment:
See also isequal is redundant with inline mentions and see also with isunordered is questionable because `isunordered` is defined by `<`. Perhaps add this to the `<` docstring.

Reply:
Good point, I've pushed a commit to do as you propose.